### PR TITLE
Add ninja to injective

### PR DESCRIPTION
**Pull request by DocBadTouch**
## Description
Added NINJA token from Injective

### Adding Assets

If adding a new asset, please ensure the following:
- [x] Asset is registered to the [Cosmos Chain Registry](https://github.com/cosmos/chain-registry).
- [x] Add asset to bottom of `zone_assets.json`.
   - The IBC channel referenced in `path` must be registered to the Chain Registry.
   - `osmosis_unlisted` defaults to `true` (until the respesentation and transferring of the new asset has been validated)

### On-chain liquidity

For each new asset, please provide the plan for on-chain liquidity of the asset: (choose one)
- [ ] Ready -- A liquidity pool has been created. The pool ID is: ______
- [x] Soon -- A pool will be created. (See: [Pool Setup Guilde](https://docs.osmosis.zone/overview/integrate/pool-setup).)
  - [x] (optional) A preview of the Osmosis Zone app with the new asset added is requested for creating the pool. (Supercharged Liquidity pools cannot be created via Osmosis Zone app)
- [ ] StreamSwap -- The token is, or will be, going through a StreamSwap stream; thus, the token should be listed without requiring on-chain liquidity. A Pool ID will be provided in this PR following the stream's completion once the team has had a chance to create a pool using the earned funds.

### CODE REVIEW

**Comment by sunnya97** [comment]
@DocBadTouch NINJA needs to be added to the Cosmos Chain Registry

https://github.com/cosmos/chain-registry/blob/master/injective/assetlist.json

**Comment by DocBadTouch** [comment]
@sunnya97 Sorry about that, I put the PR for adding everything to Cosmos Chain Registry before this one and thought it would be merged by now since other ones after mine got merged, but I guess not. I will see if I can get that PR pushed through so this can pass the validation checks.

File: osmosis-1/osmosis-1.assetlist.json [hunk 1]
```diff
@@ -12756,6 +12756,48 @@
...
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/ninja.png"
+      },
+      "keywords": [
+        "osmosis-info",
+        "osmosis-price:thiswillbethepooladdress"
+      ]
```
Comment (sunnya97) [inline osmosis-1/osmosis-1.assetlist.json:12800]:
```suggestion
```

Comment (sunnya97) [reply osmosis-1/osmosis-1.assetlist.json:12800]: This will get auto-generated by our system.

**Pull request merged**